ElectronicObserver/Window/Wpf/ShipTrainingPlanner/ShipTrainingPlanView.xaml

```diff
@@ -0,0 +1,263 @@
+				Grid.ColumnSpan="2"
+				Orientation="Horizontal"
+				>
+				<TextBlock
+					Margin="0 0 5 0"
+					VerticalAlignment="Center"
+					Text="{Binding ShipTrainingPlanner.Current}"
+					/>
+				<TextBlock VerticalAlignment="Center" Text="{Binding Ship.HPMax}" />
+				<TextBlock
+					Margin="5 0"
+					VerticalAlignment="Center"
+					Text="/"
+					/>
+				<TextBlock VerticalAlignment="Center" Text="{Binding TargetHP}" />
+			</StackPanel>
+
+			<TextBlock
+				Grid.Row="4"
+				Grid.Column="0"
+				VerticalAlignment="Center"
+				Text="{Binding ShipTrainingPlanner.ASWBonus}"
+				/>
+
+			<StackPanel
+				Grid.Row="4"
+				Grid.Column="1"
+				Orientation="Horizontal"
+				>
+				<TextBlock VerticalAlignment="Center" Text="{Binding Ship.ASWModernized}" />
+				<TextBlock
+					Margin="5 0"
+					VerticalAlignment="Center"
+					Text="/"
+					/>
+			</StackPanel>
+
+			<ui:NumberBox
+				Grid.Row="4"
+				Grid.Column="2"
+				Margin="5"
+				VerticalAlignment="Center"
+				Maximum="{Binding Ship.MasterShip.ASWModernizable}"
+				Minimum="{Binding Ship.ASWModernized}"
+				SpinButtonPlacementMode="Inline"
+				Value="{Binding TargetASWBonus}"
+				/>
+
+			<StackPanel
+				Grid.Row="5"
+				Grid.Column="1"
+				Grid.ColumnSpan="2"
+				Orientation="Horizontal"
+				>
+				<TextBlock
+					Margin="0 0 5 0"
+					VerticalAlignment="Center"
+					Text="{Binding ShipTrainingPlanner.Current}"
+					/>
+				<TextBlock VerticalAlignment="Center" Text="{Binding Ship.ASWBase}" />
+				<TextBlock
+					Margin="5 0"
+					VerticalAlignment="Center"
+					Text="/"
+					/>
+				<TextBlock VerticalAlignment="Center" Text="{Binding TargetASW}" />
+			</StackPanel>
+
+			<TextBlock
+				Grid.Row="6"
+				Grid.Column="0"
+				VerticalAlignment="Center"
+				Text="{Binding ShipTrainingPlanner.Luck}"
+				/>
+
+			<StackPanel
+				Grid.Row="6"
+				Grid.Column="1"
+				Orientation="Horizontal"
+				>
+				<TextBlock VerticalAlignment="Center" Text="{Binding Ship.LuckBase}" />
+				<TextBlock
+					Margin="5 0"
+					VerticalAlignment="Center"
+					Text="/"
+					/>
+			</StackPanel>
+
+			<ui:NumberBox
+				Grid.Row="6"
+				Grid.Column="2"
+				Margin="5"
+				VerticalAlignment="Center"
+				Maximum="{Binding Ship.MasterShip.LuckMax}"
+				Minimum="{Binding Ship.LuckBase}"
+				SpinButtonPlacementMode="Inline"
+				Value="{Binding TargetLuck}"
+				/>
+
+			<TextBlock
+				Grid.Row="7"
+				Grid.Column="0"
+				VerticalAlignment="Center"
+				Text="{Binding ShipTrainingPlanner.RemodelGoal}"
+				/>
+
+			<ComboBox
+				Grid.Row="7"
+				Grid.Column="1"
+				Grid.ColumnSpan="2"
+				Margin="5"
+				VerticalAlignment="Center"
+				DisplayMemberPath="NameEN"
+				ItemsSource="{Binding PossibleRemodels}"
+				SelectedItem="{Binding TargetRemodel}"
+				/>
+
+			<CheckBox
+				Grid.Row="8"
+				Content="{Binding ShipTrainingPlanner.NotifyTargetRemodelReady}"
+				IsChecked="{Binding NotifyOnRemodelReady}"
+				/>
+			<CheckBox
+				Grid.Row="10"
```

Comment:
This should probably be 9.
You should preferably add `Grid.Column="0"` even if it's not needed.
Add column span like the ship image.